specification/healthdataaiservices/data-plane/HealthDataAIServices.Deidentification/preview/2024-01-31-preview/HealthDataAIServices.Deidentification.json

```diff
@@ -0,0 +1,1309 @@
+          "type": "string",
+          "description": "Checksum of the file."
+        }
+      },
+      "required": [
+        "entities",
+        "stringIndexType"
+      ]
+    },
+    "SourceStorageLocation": {
+      "type": "object",
+      "description": "Storage location.",
+      "properties": {
+        "location": {
+          "type": "string",
+          "format": "uri",
+          "description": "URL to storage location. Must be a valid Azure Storage SAS URI."
+        },
+        "prefix": {
+          "type": "string",
+          "description": "Prefix to filter files by."
+        },
+        "delimiter": {
+          "type": "string",
+          "description": "Delimiter of the storage paths.",
+          "default": "/"
+        },
+        "recursive": {
+          "type": "boolean",
+          "description": "Recursive flag to indicate if all files in the prefix should be included. Only available as source.",
+          "default": true
+        },
+        "extensions": {
+          "type": "array",
+          "description": "List of extensions to filter files by. Only available as source.",
+          "default": [
+            "*"
+          ],
+          "minItems": 1,
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "location",
+        "prefix",
+        "delimiter",
+        "recursive"
+      ]
+    },
+    "StringIndexType": {
+      "type": "string",
+      "description": "List of supported index encodings.",
+      "enum": [
+        "TextElement_v8",
+        "UnicodeCodePoint",
+        "Utf16CodeUnit"
+      ],
+      "x-ms-enum": {
+        "name": "StringIndexType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "TextElement_v8",
+            "value": "TextElement_v8",
+            "description": "Iterates over graphemes as defined by the Unicode 8.0.0 standard."
+          },
+          {
+            "name": "UnicodeCodePoint",
+            "value": "UnicodeCodePoint",
+            "description": "Iterates over Unicode Code Points, the default scheme for Python 3."
+          },
+          {
+            "name": "Utf16CodeUnit",
+            "value": "Utf16CodeUnit",
+            "description": "Iterates over UTF-16 Code Units, the default scheme for JavaScript, Java, and .NET."
+          }
+        ]
+      }
+    },
+    "TargetStorageLocation": {
+      "type": "object",
+      "description": "Storage location.",
+      "properties": {
+        "location": {
+          "type": "string",
+          "format": "uri",
+          "description": "URL to storage location. Must be a valid Azure Storage SAS URI."
+        },
+        "prefix": {
+          "type": "string",
+          "description": "Prefix to filter files by."
+        },
+        "delimiter": {
+          "type": "string",
+          "description": "Delimiter of the storage paths.",
+          "default": "/"
+        }
```

Comment:
"delimiter" is required and cannot have a default

Reply:
Removed.